test/e2e/example_lazy_test.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package e2e
+
+import (
```

Comment:
We do have `require` package as a dependency, and we are using it to do all the assertions in the other tests, you can see an example [here](https://github.com/carvel-dev/vendir/blob/develop/test/e2e/github_release_test.go).
Do you mind refactoring this test to make it similar to the other tests?

Reply:
Not at all - look much more concise now